### PR TITLE
Remove unused flags, add description

### PR DESCRIPTION
**Pull request by LeoKuhorev**
- [x] remove not applicable cmdlet parameters
- [x] add more detailed example description

### CODE REVIEW

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@get-itips @dariomws @tseward

File: teams/teams-ps/teams/Get-CsTeamsMediaLoggingPolicy.md [hunk 1]
```diff
@@ -21,14 +21,13 @@ Returns information about the Teams Media Logging policy.
 ### Identity (Default)
 
 ```
-Get-CsTeamsMediaLoggingPolicy [[-Identity] <String>] [-MsftInternalProcessingMode <String>]
- [<CommonParameters>]
+Get-CsTeamsMediaLoggingPolicy [[-Identity] <String>] [<CommonParameters>]
```
Comment (dariomws) [inline teams/teams-ps/teams/Get-CsTeamsMediaLoggingPolicy.md:24]: Please also remove the parameter from the Parameters section.

File: teams/teams-ps/teams/Grant-CsTeamsMediaLoggingPolicy.md [hunk 2]
```diff
@@ -22,23 +22,16 @@ Assigns Teams Media Logging policy to a user, group of users or the entire tenan
...
 
 ### GrantToTenant
 
 ```
-Grant-CsTeamsMediaLoggingPolicy [-PassThru] [[-PolicyName] <String>] [-MsftInternalProcessingMode <String>]
+Grant-CsTeamsMediaLoggingPolicy [-PassThru] [[-PolicyName] <String>]
  [-Global] [-WhatIf] [-Confirm] [<CommonParameters>]
 ```
 
-### GrantToGroup
```
Comment (dariomws) [inline teams/teams-ps/teams/Grant-CsTeamsMediaLoggingPolicy.md:35]: It seems that Group and Rank parameters are available. Why remove them?

**Review by dariomws — CHANGES_REQUESTED** [review]

**Comment by dariomws** [comment]
@LeoKuhorev, If you have a Microsoft alias, you can link your accounts in GitHub so that a blue "Microsoft submitter" label appears to any PR's or issue you submit. Guidance on it here: https://docs.opensource.microsoft.com/tools/github/index.html

Here is the site you can use to see the mappings between GitHub and Microsoft aliases: [https://repos.opensource.microsoft.com/people?q=<alias>](https://repos.opensource.microsoft.com/people?q=LeoKuhorev)

**Review by get-itips — CHANGES_REQUESTED** [review]
Also agree that if the parameter is still in the code we need to keep it in the documentation otherwise people open issues when they find it (I don't see Group and Rank in 4.6.1-preview but I do see MsftInternalProcessingMode)

File: teams/teams-ps/teams/Get-CsTeamsMediaLoggingPolicy.md (hunk 1 above)
Comment (LeoKuhorev) [reply teams/teams-ps/teams/Get-CsTeamsMediaLoggingPolicy.md:24]: Thanks for catching this! Fixed

File: teams/teams-ps/teams/Grant-CsTeamsMediaLoggingPolicy.md (hunk 2 above)
Comment (LeoKuhorev) [reply teams/teams-ps/teams/Grant-CsTeamsMediaLoggingPolicy.md:35]: I checked `-Group` and `-Rank` parameters with this cmdlet as well as some other Grant-<..> cmdlets, and it seems that those are not implemented as I'm getting 'A parameter cannot be found' error.

**Comment by LeoKuhorev** [comment]
> Also agree that if the parameter is still in the code we need to keep it in the documentation otherwise people open issues when they find it.

It seems like those parameters are not implemented in neither this nor the other Grant-<...> cmdlets. And also they're not listed in other Grant-<..> cmdlets docs, so removing them here as well for consistency

**Comment by get-itips** [comment]
@LeoKuhorev 
Can you kindly specify against which version are you testing the existence of the parameters? Many thanks Leo

**Comment by LeoKuhorev** [comment]
Initially I was using 4.5.0. I've just installed 4.6.1-preview, and it seems to be able to find the `-Group` parameter, but now it looks like TeamsMediaLoggingPolicy is not enabled in flighting for the Group Policy Assignment. Let me see if enabling it in flighting will resolve the issue. Thanks!

**Comment by yogkumgit** [comment]
@get-itips @dariomws @tseward

**Comment by get-itips** [comment]
@yogkumgit please see https://github.com/MicrosoftDocs/office-docs-powershell/pull/9854#issuecomment-1233468609 as we are now waiting on some checks by @LeoKuhorev

**Comment by LeoKuhorev** [comment]
After the flighting I confirmed that both `-Group` and `-Rank` parameters are working fine, so I'm bringing them back to docs as well as introducing a new example for the group policy assignment. Please take a look when you get a chance. Thanks!

**Comment by get-itips** [comment]
@LeoKuhorev It would be good to bring those back in the same line numbers that were before, this helps later when looking at the file history (to understand nothing actually changed about those parameters)
Thanks for checking that

**Review by dariomws — APPROVED** [review]

**Comment by dariomws** [comment]
I agree with @get-itips, it would be great to have it.

**Review by tseward — APPROVED** [review]

**Comment by LeoKuhorev** [comment]
Want to provide some updates on group policy assignment. I sync'd with the infra team and it occurred that currently group assignment is fully available for 12 policies listed in the note section [here](https://docs.microsoft.com/en-us/powershell/module/teams/new-csgrouppolicyassignment?view=teams-ps#description). For other policies group assignment is only available for internal and TAP users. There's currently work going to extend the availability to general public.

Therefore, I'm removing reference to `-Group` and `-Rank` flags for now and will bring them back once group policy assignment is available for everyone. 

Please review and let me know if you have any questions. Thanks!

**Comment by yogkumgit** [comment]
@get-itips

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**